returnn/datasets/lm.py

```diff
@@ -178,26 +181,26 @@ def __init__(self,
     if word_end_symbol and not word_based:  # Character-based modeling and word_end_symbol is specified.
       self.orth_replace_map[" "] = [word_end_symbol]  # Replace all spaces by word_end_symbol.
 
-    num_labels = len(self.labels["data"])
```

Comment:
I would keep the old variable name `num_labels`.

Reply:
I do not like it but ok.